components/email/src/Email.svelte

```diff
@@ -534,50 +534,50 @@
       message: message,
       type,
     });
+    const subject = message.subject?.toLowerCase().startsWith("re:")
+      ? message.subject
+      : `Re: ${message.subject}`;
+    const value = {
+      reply_to_message_id: message.id,
+      from,
+      to,
+      reply_to: from,
+      cc,
+      bcc: message.bcc,
+      body: message.body,
+      subject: subject,
+    };
+
+    let event_identifier;
+    switch (type) {
+      case "reply":
+        event_identifier = "replyClicked";
+        break;
+
+      case "reply_all":
+        event_identifier = "replyAllClicked";
+        break;
+    }
 
     //Check existing draft
     const existingDraft = activeThread?.drafts?.find(
       (draft) => draft.reply_to_message_id === message.id,
     );
     if (existingDraft) {
-      existingDraft.to = to;
-      existingDraft.cc = cc;
-      dispatchDraftEvent(event, existingDraft);
+      value.body = existingDraft.body;
```

Comment:
Where are you getting the body from? `activeThread` messages and drafts don't contain the message body AFAIK. You have to use `fetchIndividualMessage` to get the body prop